src/pages/Signup.jsx

```
@@ -1,123 +1,189 @@
+      });
+
+      setName('');
+      setEmail('');
+      setPhone('');
+      setPassword('');
+      setPasswordConfirmation('');
+      setErrors({});
+      navigate('/login');
+    } catch (e) {
+      if (e.response && e.response.data && e.response.data.errors) {
+        setErrors(e.response.data.errors);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)